Pathogens/Utility/location_utils/lat_lng_generation_report.py

```diff
@@ -0,0 +1,314 @@
+from enum import Enum
```

Comment:
Alright this file is a bit hard to explain but I'll try. If you're confused at all it straight up might be better to talk in person so just send me a DM and we can hop on a call or something.

All of my `generate_line`-type functions with the exception of `generate_line_for_response_and_request` print out a line if they flag something that I've seen before that I know can be an issue for our data generation.

I know this function `WARN`s on a fair amount of false positives for bad data. My idea here is to autogenerate the report every so often and skim it for anything that seems serious and if I think it seems serious enough we go in and fix it. It's worth noting that almost every data point that actually causes a problem actually logs an `ERROR` and the `ERROR`s are almost all true positives for bad data.

I'm going to try to annotate this the best I can but sorry if it's still confusing despite that. Again, I'm super open to questions it's just a bit hard to explain after how far I've gone down the rabbit hole lol

I feel like reading one of the reports and seeing if you can follow along might help
[lat_lng_generation_report.txt](https://github.com/serotracker/iit-backend/files/13667486/lat_lng_generation_report.txt)

I think there's room to iterate on this but I'm pretty happy with how it turned out all things considered.